questionpy_sdk/commands/run.py

```diff
@@ -47,7 +48,24 @@ async def async_run(pkg_location: PackageLocation, state_storage_path: Path, hos
     "--port", "-p", "port", default=8080, show_default=True, type=click.IntRange(1024, 65535), help="Port to bind to."
 )
 @click.option("--watch", "-w", "watch", is_flag=True, help="Watch source directory and rebuild on changes.")
-def run(package: str, state_storage_path: Path, host: str, port: int, *, watch: bool) -> None:
+@click.option(
+    "--worker",
+    "-W",
+    "worker",
+    type=click.Choice(("subprocess", "thread"), case_sensitive=False),
```

Comment:
Hier bietet sich vielleicht ein eigener Parameter-Typ an, der direkt die Konvertierung übernimmt, [siehe](https://click.palletsprojects.com/en/stable/parameter-types/#how-to-implement-custom-types). (Dann bäuchte man das auch nicht mehr unten (Zeile 86) machen.)

Reply:
Da hatte ich tatsächlich auch mit begonnen, stellte sich aber als wesentlich aufwändiger als das hier heraus. Falls wir so eine Option irgendwann mehrmals haben, können wir die Idee ja noch revisiten